### PR TITLE
feat: add seasonal halloween theme

### PR DESCRIPTION
**Pull request by 4444dogs**
## Please make sure to check the following tasks before opening and submitting a PR

* [x] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
* [x] I have tested my changes locally and they are working as intended
* [x] These changes do not have any notable side effects on other Revolt projects
* [ ] (optional) I have opened a pull request on [the translation repository](https://github.com/revoltchat/translations)
* [x] I have included screenshots to demonstrate my changes

This PR just adds Halloween emojis using the code for the Christmas snowflakes in October.

Screenshots:
image

### CODE REVIEW

File: src/pages/home/Home.tsx [hunk 1]
```diff
@@ -51,24 +51,40 @@ export default observer(() => {
...
+                flakes.push("👻");
+                flakes.push("⚰️");
+                flakes.push("🕷️");
+            }
+
+            return flakes;
         }
 
-        return flakes;
+        else return [];
```
Comment (Rexogamer) [inline src/pages/home/Home.tsx:87]: This can remain as is - if `isDecember` and `isOctober` are both false, the `flakes` array will still be empty

**Review by Rexogamer — CHANGES_REQUESTED** [review]
Minor code nitpick; other than that, LGTM

File: src/pages/home/Home.tsx (hunk 1 above)
Comment (4444dogs) [reply src/pages/home/Home.tsx:87]: Alright, I just changed `if` to `else if` so that `else` applies to both cases. (should fix that little issue)

Comment (Rexogamer) [reply src/pages/home/Home.tsx:87]: I think you slightly misunderstood my comment - I meant that none of the `else`s are required

**Pull request merged**